### ISSUE TITLE
Wrong album title using YouTube Music

### ISSUE THREAD

**Issue by anggiedwarsa** [simple-last-fm-scrobbler/sls#571]
Album title is wrong with Youtube music. Simple Scrobble choose the previous song's album. Not curently playing song's album. Please fix. :(

**Comment by lexxsoft** [comment]
I hope no one would object to bump this ticket up.
As far as I can add, this problem still persists, on multiple devices, and even on Youtube Music Vanced app.

Has anyone investigated this, any ideas why the metadata is taken from previous track?
On the other hand, maybe someone could suggest alternative scrobbler, because the official has a lot of hate on google play for missed scrobbles.

**Comment by RezurrectBodg** [comment]
@lexxsoft It doesn't lose a single track if you disable battery optimizations, and not a single living soul on play market does that, unfortunately.

**Comment by lexxsoft** [comment]
> @lexxsoft It doesn't lose a single track if you disable battery optimizations, and not a single living soul on play market does that, unfortunately.

No, I don't believe that is a case here. I have explicitly allowed the application to function in a background, and I am not losing any scrobbles. However the album name is incorrect - it is always read from previously played track, everything else seems to work fine.

I have temporarily switched to last.fm's scrobbler, and it does work correctly, though I really don't like its lack of customization and ability to submit data to third party services.

**Comment by jonpsp** [comment]
I have the same problem when using the YouTube Music app on my Android 10 (LineageOS 17.1) phone (track name and artist are correct, but album is for the previous track). I have not noticed any missed scrobbles, but scrobbles based on using YouTube in my web browser don't appear at last.fm (I wonder if this is because they don't have an album name?)

**Comment by RezurrectBodg** [comment]
@lexxsoft yes, i guess it may be messing up titles in yt music, but I don't use that. I was talking about missed scrobbles, and I haven't those.

**Comment by RezurrectBodg** [comment]
Also keep in mind that many OEMs add another optimization layer to the default android optimization list, so there are basically two of them.
